content/applications/inventory_and_mrp/inventory/product_management/product_replenishment/uom.rst

```diff
@@ -60,55 +58,68 @@ Specify a product's units of measure
+at the top-right corner of the |PO|. Odoo automatically converts the purchase unit of measure into
+the product's sales/inventory unit of measure, so the :guilabel:`Demand` column of the delivery
+receipt shows the converted quantity.
 
-For example, if the product's purchase UoM is `Box of 6` and its sales/inventory unit of measure is
-`Units`, the PO shows the quantity in boxes of six, and the delivery receipt shows the quantity in
-units.
+For example, if the product's purchase |UOM| is `Box of 6` and its sales/inventory unit of measure
+is `Units`, the |PO| shows the quantity in boxes of six, and the receipt (and other internal
+warehouse documents) shows the quantity in units.
 
 Replenishment
 -------------
 
 A request for quotation for a product can also be generated directly from the product form using
-the :guilabel:`Replenish` button. After clicking :guilabel:`Replenish`, a replenish assistant box
-pops up. The purchase unit of measure can be manually edited here if needed. Then, click
-:guilabel:`Confirm` to create the RFQ.
+the :guilabel:`Replenish` button.
+
+After clicking :guilabel:`Replenish`, a replenish assistant box pops up. The purchase unit of
+measure can be manually edited in the :guilabel:`Quantity` field, if needed. Then, click
+:guilabel:`Confirm` to create the |RFQ|.
+
+.. important::
+   A |PO| can only be automatically generated if at least one vendor is listed in the product form's
+   :guilabel:`Purchase` tab.
+
+.. image:: uom/replenish.png
+   :align: center
+   :alt: Click Replenish button to manually replenish.
 
-Next, click the :guilabel:`Units Forecasted` smart button on the product form and scroll down to
-:menuselection:`Forecasted Inventory --> Requests for quotation`. Click on the RFQ reference number
-to open the draft RFQ. The purchase UoM can also be edited here if needed.
+Navigate to the created |PO| by clicking the :guilabel:`Forecasted` smart button on the product
+form. Scroll down to :guilabel:`Forecasted Inventory` section, and in the :guilabel:`Requests for
+quotation` line, click the |RFQ| reference number to open the draft |RFQ|. If necessary, the
+purchase |UOM| can be edited directly on the |PO|.
 
 Sell in a different UoM
 -----------------------
 
-When creating a new quotation in the Sales app, Odoo automatically uses the product's specified
-unit of measure. However, if needed, the :guilabel:`UoM` can be manually edited on the quotation.
+When creating a new quotation in the *Sales* app, Odoo automatically uses the product's specified
+unit of measure. If needed, the :guilabel:`UoM` can be manually edited on the quotation.
 
-After the quotation is sent to the customer and confirmed into a sales order (SO), click on the
-:guilabel:`Delivery` smart button at the top right corner of the SO. Odoo automatically converts
+After the quotation is sent to the customer and confirmed into a sales order (SO), click the
```

Comment:
Do you think an image might add to this section? Not entirely necessary

Reply:
Added a full example, with figures !

Reply:
I'm not sure I'm seeing the image here, if it was added